loops-arrays-objects.js

```diff
@@ -48,21 +61,40 @@ const scores = [
 //  { name: 'Rianne', score: 66, grade: 'D' }
 //  ];
 // ==========================================
-
-
-
+// input = de array heeft nu bij elk object met property grades null staan.
+// output = bij grades staan nu letters die corresponderen met hun score.
+// stappenplan:
+// 1. bij opdracht 1 B zijn de scores al omgerekend naar de grade-letter
+// 2. pas voor elk object de grade waarde aan met de juiste grade letter.
+
+for (let i = 0; i < scores.length; i++) {
+    if (scores[i].score < 60) {
+        scores[i].grade = 'F';
+    } else if (scores[i].score >= 60 && scores[i].score < 70) {
+        scores[i].grade = 'D';
+    } else if (scores[i].score >= 70 && scores[i].score < 80) {
+        scores[i].grade = 'C';
+    } else if (scores[i].score >= 80 && scores[i].score < 90) {
+        scores[i].grade = 'B';
+    } else if (scores[i].score >= 90 && scores[i].score <= 100) {
+        scores[i].grade = 'A';
+    } else {
+        console.error('er gaat iets mis')
+    }
+}
+console.log(scores);
 
```

Comment:
Vergelijkbaar met de vorige opdracht. Overbodige check in de conditie van de if-statement